src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/base_js.html.twig

```diff
@@ -235,6 +235,46 @@
         }
 
         {% if excluded_ajax_paths is defined %}
+
+            if (window.fetch) {
+                var oldFetch = window.fetch;
+                window.fetch = function () {
+                    if (!arguments[0].match(new RegExp({{ excluded_ajax_paths|json_encode|raw }}))) {
+
+                        var promise = oldFetch.apply(null, arguments);
+
+                        var method = 'GET';
+                        if(arguments[1].method !== undefined){
```

Comment:
@ivoba The `init` config object is optional (`Promise<Response> fetch(input[, init]);`) therefore this can throw an error. First check if the 2nd argument even exists `arguments[1] && arguments[1].method !== undefined` .